index.js

```diff
@@ -177,6 +185,10 @@ function _make_generateLog(emitter, func, start_time, config, context) {
           client_id: config.clientId
         }
 
+        if (COLDSTART) {
```

Comment:
Making sure I understand the idea: so when the agent is loaded, COLDSTART will exist on the machine; if it's there, we set COLDSTART to 0? If that's so, shouldn't the definition be `COLDSTART = COLDSTART || 1`?